py3-sqlalchemy-cockroachdb.yaml

```diff
@@ -0,0 +1,38 @@
+# Generated from https://pypi.org/project/sqlalchemy-cockroachdb/
+package:
+  name: py3-sqlalchemy-cockroachdb
+  version: 2.0.1
+  epoch: 0
+  description: CockroachDB dialect for SQLAlchemy
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-sqlalchemy
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: ddb46e2da53d812bf7b8ef21c3cafb918004747f815a283f374796bf57a9497a
+      uri: https://files.pythonhosted.org/packages/72/74/7157d36dee9175864f803fdbc931913dac17b0d4eee24727dbdac1244543/sqlalchemy-cockroachdb-${{package.version}}.tar.gz
```

Comment:
Is there a reason we're not pulling this from git?
- https://github.com/cockroachdb/sqlalchemy-cockroachdb

I also noticed that the latest tag in Git is v2.0.0, which is strange - there doesn't seem to be a tag cut for 2.0.1

Reply:
Yep that's why I didn't use GitHub here.